### PR TITLE
Add cinema: The Light Cinemas

### PR DESCRIPTION
**Pull request by davidferguson**
Adds [The Light Cinemas]() to the archiver.

Note: as far as I can see, The Light does not display the release year of movies _anywhere_, and therefore **this archiver does not add release years**, and simply fills them as `0`.

Also note: Light Cinemas is interesting. They have two domains, lightcinemas.co.uk and thelight.co.uk, and some of their cinemas are subdomains of the first, and others the second (never both). The API is the same for both, but the booking system is different. We use the booking system to get the screen of a showing, so there is two functions for getting the screen, one for each of the booking systems.

### CODE REVIEW

**Comment by andrewferguson** [comment]
For the CentreForTheMovingImage (Filmhouse) archiver, I set the year to "" (empty string) when a year couldn't be found. I'm not sure if there's a particular reason for choosing one or the other, but we should decide on a standard before merging this PR.

**Comment by davidferguson** [comment]
I'm now using `''` for both, which is defined in the `consts.py` file.

**Comment by davidferguson** [comment]
This now also includes the archiver for [Parkway Cinemas](https://parkwaycinemas.co.uk) since most of their websites are the same as The Light. But like The Light, they're also special and one cinema has its own website which is quite similar to ARC/Merlin. So the code is a little messy now. Could maybe be refactored a bit once things are merged.

**Review by andrewferguson — APPROVED** [review]
Looks fine and works on my machine.

**Pull request merged**
